bundles/org.openhab.binding.electroluxappliances/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,472 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="electroluxappliances"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="api">
+		<label>Electrolux Group API</label>
+		<description>This bridge represents the web API connector.</description>
+
+		<properties>
+			<property name="vendor">Electrolux</property>
+		</properties>
+
+		<config-description>
+			<parameter name="apiKey" type="text" required="true">
+				<label>API Key</label>
+				<description>Your personal API key.</description>
+			</parameter>
+			<parameter name="accessToken" type="text" required="true">
+				<label>Access Token</label>
+				<description>Your personal Access Token.</description>
+			</parameter>
+			<parameter name="refreshToken" type="text" required="true">
+				<label>Refresh Token</label>
+				<description>Your personal Refresh Token.</description>
+			</parameter>
+			<parameter name="refresh" type="integer" min="10" unit="s">
+				<label>Refresh Interval</label>
+				<description>Specifies the refresh interval in seconds.</description>
+				<default>300</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="electroluxairpurifier">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="api"/>
+		</supported-bridge-type-refs>
+
+		<label>ElectroluxAir Purifier</label>
+		<description>This thing represents the Electrolux Air Purifier.</description>
+
+		<channels>
+			<channel id="temperature" typeId="temperature"/>
+			<channel id="humidity" typeId="humidity"/>
+			<channel id="tvoc" typeId="tvoc"/>
+			<channel id="pm1" typeId="pm1"/>
+			<channel id="pm2_5" typeId="pm2_5"/>
+			<channel id="pm10" typeId="pm10"/>
+			<channel id="co2" typeId="co2"/>
+			<channel id="filterLife" typeId="filterLife"/>
+			<channel id="doorOpen" typeId="doorOpen"/>
+			<channel id="fanSpeed" typeId="fanSpeed"/>
+			<channel id="workMode" typeId="workMode"/>
+			<channel id="ionizer" typeId="ionizer"/>
+			<channel id="uiLight" typeId="uiLight"/>
+			<channel id="safetyLock" typeId="safetyLock"/>
+			<channel id="status" typeId="status"/>
+			<channel id="connectionState" typeId="connectionState"/>
+		</channels>
+
+		<properties>
+			<property name="vendor">Electrolux</property>
+			<property name="thingTypeVersion">1</property>
```

Comment:
Initial bindings do not have a thing type version
```suggestion
```

Reply:
Fixed"